src/gui_common/tooltip/ToolTipManager.cs

```diff
@@ -516,4 +518,32 @@ private void HideAllToolTips()
             }
         }
     }
+
+    private void CreateOrganelleTooltips()
+    {
+        if (SimulationParameters.Instance == null)
+            return;
```

Comment:
This should be a really loud error, arguably this should maybe prevent the game from starting up as this would result in a really broken game if this failed.